app/api/v1/models/election.py

```diff
@@ -100,6 +106,23 @@ class ElectionQueryResponse(BaseResponse):
     elections: List[Election] = []
 
 
+class ElectionSummaryDto(Base):
+    election_id: str
+    name: Any
+    state: str
+    number_of_guardians: int
+    quorum: int
+    cast_ballots: int
```

Comment:
The ballot inventory uses cast_ballot_count and spoiled_ballot_count.  Should these be changed to be consistent?

Reply:
yup, good catch